src/Maker/MakeEntity.php

```diff
@@ -619,6 +619,20 @@ function ($name) use ($targetClass) {
             $relation->setMapInverseRelation($mapInverse);
         };
 
+        $askJoinTableName = function (EntityRelation $relation) use ($io) {
+            $joinTableDecision = $io->confirm(
+                'Do you want to specify a join table? You may want to do this if you plan on having multiple many-to-many relations to the same entity.',
```

Comment:
Open to suggestions on rewording

Reply:
In the current placement, this question will be asked for all relation types, right? But we only want it for `ManyToMany`

For the wording, what about just one question:

> A join table will be created for the relationship. You can name this, or let Doctrine choose a name for you.
> Join table name (default: set automatically): 

If they hit enter (leave it blank), then we know to name it automatically.

Reply:
This area just defines and stores the function as a variable. The logic within isn't actually ran until the `$askJoinTableName($relation);` statement is executed later down in the code [here](https://github.com/symfony/maker-bundle/pull/1416/files#diff-dc199e4a5d406f1d06cbb6b936027bccbbaf2e4e1ec7be7d5f54bd6393fa1955R725).

As much as I'd prefer the lower complexity of one question, we'll want to give the user the ability to say no to the addition of a join table. Cases like [bidirectional many-to-many](https://www.doctrine-project.org/projects/doctrine-orm/en/2.17/reference/association-mapping.html#many-to-many-bidirectional) and [self referencing many-to-many](https://www.doctrine-project.org/projects/doctrine-orm/en/2.17/reference/association-mapping.html#many-to-many-self-referencing) require the use of a ManyToMany attribute without a JoinTable attribute on one side.